deps.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+#--- Script defining values to be used by accompanying install scripts ---#
+
+
+### Setup ###
+
+## Source the default values for imported variables
+if [ -z "${working_dir}" ]; then
+    . scripts/default_values.sh
+else
+    ## 'working_dir' is not empty => Script was called from anonymous shell
+    . "${working_dir}/scripts/default_values.sh"
+fi
+
+## Apt
+export script_path_apt="scripts/install_apt_deps.sh"
+
+## OpenCV
+export dep_name_opencv="opencv"
+export script_path_opencv="scripts/install_opencv.sh"
+export parent_dir_opencv="${opt_dir}"
+export dep_prompt_opencv="Install OpenCV from source"
+export dep_ver_opencv="3.4.6-instrumented"
+
+## Eigen
+export dep_name_eigen="eigen"
+export script_path_eigen="scripts/install_eigen.sh"
+export parent_dir_eigen="${opt_dir}"
+export dep_prompt_eigen="Install eigen (Eigen3) from source"
+export dep_ver_eigen="3.3.7"
```

Comment:
Not 3.3.9?

Reply:
Version `3.3.7` matches the current version installed by Ubuntu 20.04 LTS. Should it be `3.3.9` for everyone? I forgot which plugin requires it. KinectFusion?

Reply:
Kimera needs 3.3.9 otherwise we have to use Kimera's built-in Eigen. Isn't that the entire fix that you pushed a few weeks ago?